buildSrc/version.properties

```diff
@@ -3,29 +3,31 @@ elasticsearch     = 6.0.0-alpha1
 lucene            = 6.5.0-snapshot-f919485
 
 # optional dependencies
-spatial4j         = 0.6
-jts               = 1.13
+guava             = 18.0
 jackson           = 2.8.6
-snakeyaml         = 1.15
+jimfs             = 1.1
+jna               = 4.2.2
+jts               = 1.13
 # When updating log4j, please update also docs/java-api/index.asciidoc
 log4j             = 2.7
 slf4j             = 1.6.2
-jna               = 4.2.2
+snakeyaml         = 1.15
+spatial4j         = 0.6
 
 # test dependencies
-randomizedrunner  = 2.4.0
-junit             = 4.11
+commonscodec      = 1.10
+commonslogging    = 1.1.3
+hamcrest          = 1.3
+# When updating httpasyncclient, please also update core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
+httpasyncclient   = 4.1.2
 httpclient        = 4.5.2
 # When updating httpcore, please also update core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
 httpcore          = 4.4.5
-# When updating httpasyncclient, please also update core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
-httpasyncclient   = 4.1.2
-commonslogging    = 1.1.3
-commonscodec      = 1.10
-hamcrest          = 1.3
-securemock        = 1.2
+junit             = 4.11
 # When updating mocksocket, please also update core/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy
 mocksocket        = 1.1
+randomizedrunner  = 2.4.0
+securemock        = 1.2
```

Comment:
Sorting this file kind of broke the `# optional dependencies` and `#test dependencies` comments. I'd be fine with sorting the subsections, though it is probably a good idea to separate that from the Guava question.